xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -346,10 +440,18 @@
         <Parameter Name="unmanagedDllName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="unmanagedDllName">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="unmanagedDllName">Name of the native unmanaged DLL to be loaded without its extension</param>
+        <summary>Loads an unmanaged DLL by simple name.</summary>
+        <returns>Pointer to the loaded DLL</returns>
```

Comment:
For consideration: Maybe we should not be using the term "DLL" here, as it has typically a platform specific meaning (Windows only). Maybe we should stick to the term "library". I know the API has the "DLL" in the name, but that's something we can fix anymore.